### PR TITLE
Add 2.479.x release line, remove 2.440.x release line

### PR DESCRIPTION
**Pull request by MarkEWaite**
## Add 2.479.x release line, remove 2.440.x release line

- Add 2.479.x release line
- Remove 2.440.x line
- Replace 2.440 with 2.452 in documentation
- Document the (almost) final 2.440.x BOM version

Step in the [2.479.1 LTS release checklist](https://github.com/jenkins-infra/release/issues/606)

### Testing done

Confirmed that `mvn clean verify` and `PLUGINS=structs,mailer TEST=InjectedTest bash local-test.sh` pass locally.

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,9 +98,10 @@ When an older Jenkins version is used, then the matching older version of the pl
 | bom-2.401.x | 2745.vc7b_fe4c876fa_  |                              |
 | bom-2.414.x | 2982.vdce2153031a_0   |                              |
 | bom-2.426.x | 3208.vb_21177d4b_cd9  |                              |
+| bom-2.440.x | 3413.v0d896b_76a_30d  | Last LTS to support Java 11  |
```
Comment (jglick) [inline README.md:101]: 2.452.x and 2.462.x still support Java 11 so I am confused by this statement.

Comment (MarkEWaite) [reply README.md]: Thanks for detecting that!  My enthusiasm got the best of me and I made a false statement in that comment.

Fixed in 5092977c00577d369b26ae999257306af9e8d569

**Comment by MarkEWaite** [comment]
This is ready to review.  I recommend that we release it next week so that we have several weeks with the bom-2.479.x profile available.

**Review by jglick — APPROVED** [review]

**Review by basil — APPROVED** [review]
Thanks!

**Pull request merged**